### DISCUSSION TITLE
GithuB Foundation Certificate Free voucher not applied

### DISCUSSION THREAD

**Discussion by abdullah-dev5** [community/community#139189] [GitHub Education]
### Hi everyone,

Hey ! there i'm trying to schdule my exam for github Foundation certification for free . but when i'm schduling exam . the voucher is not applied it still shows to pay me 49$.

Help me out . how i can continue further

**Comment by davevad93** [comment]
Hi @abdullah-dev5 ,

Check the last update in "[New in the Student Developer Pack: Students, get GitHub Foundations certified!](https://github.com/orgs/community/discussions/138834#discussioncomment-10704899)", there's a bug and they have temporarily disabled the certification offer, I guess it will be available when they fix the issue.

**Comment by abdullah-dev5** [comment]
@davevad93  Maybe they change the Plan. i think they will not offer it again !

**Reply by davevad93** [reply]
I don't know @abdullah-dev5 , but the offer has been announced on September 16 and only 2 days later it has been disabled. I don't think the offer lasted only 2 days, maybe by next week there will be news, they will surely announce it in the discussion post.
